Makefile

```diff
@@ -209,3 +177,62 @@ goimports:
 ## Runs static code analysis checks (golangci-lint)
 check: gofmt goimports
 	@golangci-lint run --build-tags fuse_cli --max-same-issues 0 --verbose
+
+### k8s demos
+
+# todo: scripts to datamon-fuse-sidecar Docker img, lint to CI
+.PHONY: lint-init-scripts
+## build shell container used in fuse demo
+fuse-demo-lint-init-scripts:
+	shellcheck hack/fuse-demo/wrap_*.sh
+
+.PHONY: fuse-demo-build-shell
+## build shell container used in fuse demo
+fuse-demo-build-shell:
+	@echo 'building fuse demo container'
+	docker build \
+		--progress plain \
+		-t gcr.io/onec-co/datamon-fuse-demo-shell \
+		--ssh default \
+		-f ./hack/fuse-demo/shell.Dockerfile \
+		.
+	docker push gcr.io/onec-co/datamon-fuse-demo-shell
+
+## demonstrate a fuse read-only filesystem
+fuse-demo-ro: fuse-demo-build-shell
+	@docker image push gcr.io/onec-co/datamon-fuse-demo-shell
+	@./hack/fuse-demo/create_ro_pod.sh
+	@sleep 8 # dumb timeout on container startup
```

Comment:
Not a blocker, you can take a look at [wait](https://hackernoon.com/kubectl-tip-of-the-day-wait-like-a-boss-40a818c423ac)